packages/office-addin-project/src/commands.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as commander from "commander";
+import { logErrorMessage } from "office-addin-usage-data";
+import { usageDataObject } from "./defaults";
+import { convertProject } from "./convert";
+
+export async function convert(command: commander.Command) {
+  try {
+    const manifestPath: string = command.manifest ?? "./manifest.xml";
+
+    await convertProject(manifestPath);
```

Comment:
I think we might want to inform the developers the changes that this command will make to their project (i.e. changing the manifest and updating the project dependencies) and ask for confirmation of the changes just so they are aware of what is going to happen.  Git will help let them see the changes before committing them, but a warning might be useful.

Reply:
Sure, I just added a confirmation prompt